### ISSUE TITLE
st.load and st.data() not working in rcall 3.0.5

### ISSUE THREAD

**Issue by bbachilles** [haghish/rcall#21]
Hello, 

A recent update to rcall seems to have broken the functions for moving data from R to Stata (st.load) and Stata to R (st.data). The below code shows the issues:

```
rcall: require(lme4)
rcall: data(sleepstudy)
rcall: ls() 		
rcall: st.load(sleepstudy)
```

Leads to:

```
Warning message:
 In  :  
R failed to export Stata data set
```

Likewise, for moving data from Stata to R:

```
sysuse auto, clear
rcall: auto <- st.data()
```

Leads to:

```
Warning message:
 In  :  
Error in stata_read(filepath, missing.type, select.rows, select.cols, :  function 'Rcpp_precious_remove' not provided by package 'Rcpp'
```

Thanks!

**Comment by haghish** [comment]
I tested your code on my Mac and it runs perfectly with rcall 3.0.5 and R version 3.5.1 and readstata13 version 0.9.2. Are you sure you have installed readstata13 package properly?

**Comment by bbachilles** [comment]
I installed the latest stable version of rcall using `github install haghish/rcall, stable`. I also reinstalled readstata13 in R, but still no luck. I get the same error messages. I am on R 4.0.3 on Windows 10. My colleague also gets these errors on R 4.0.5 and Windows 10.  It's so strange because this worked fine after you fixed my prior issue (#20).

**Comment by bbachilles** [comment]
Update: I opened RStudio and restarted the R session. Once I did that, everything worked correctly using rcall within Stata. I'm not sure why this might have fixed the problem, but it is what seems to have worked.
